src/components/login/loginModel.js

```diff
@@ -22,10 +22,22 @@ class LoginModel {
     }
   }
 
-  login(){
-    let username = storage.get_username();
-    let password = storage.get_password();
-    if(view.state.username != username || view.state.password != password){
+  loginApi(){
+    fetch('https://backendpipo.herokuapp.com/login', {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      "usuario": view.state.username,
+      "contrasena": view.state.password
+    }),
+  })
+  .then(response => response.json())
+  .then(responseJson =>{
+    console.log(responseJson.profileData);
+    if(view.state.username!=responseJson.profileData.usuario){
```

Comment:
Al dar click en iniciar sesión sale un error de `undefined is not an object (evaluating 'responseJson.profileData.usuario')`